code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -135,6 +136,9 @@
 	. = ..()
 
 	emittersemicd = TRUE
+	for(var/hudtype in list(DATA_HUD_SECURITY_ADVANCED, DATA_HUD_MEDICAL_ADVANCED, DATA_HUD_DIAGNOSTIC_ADVANCED, DATA_HUD_DIAGNOSTIC_BASIC))  // no freebies go buy the huds
```

Comment:
what is this? stop the hud from being added in the first place rather than removing it. find what adds it

Reply:
it could be from any source this just helps ensure it its fine-

Reply:
this is not a proper fix, you should find what's adding it at the start and stop it. nothing else should be adding it

Reply:
it's probably caused by this
https://github.com/BeeStation/BeeStation-Hornet/pull/7351/commits/b73342075caa05cc17d0c4911fbba6433e4b6454